extensions/auth-bearer/pkg/config/config.go

```diff
@@ -1,23 +1,29 @@
 package config
 
-import "github.com/owncloud/ocis/ocis-pkg/shared"
+import (
+	"context"
+
+	"github.com/owncloud/ocis/ocis-pkg/shared"
+)
 
 type Config struct {
 	*shared.Commons `yaml:"-"`
 	Service         Service  `yaml:"-"`
 	Tracing         *Tracing `yaml:"tracing"`
-	Logging         *Logging `yaml:"log"`
+	Log             *Log     `yaml:"log"`
 	Debug           Debug    `yaml:"debug"`
-	Supervised      bool     `yaml:"-"`
 
 	GRPC GRPCConfig `yaml:"grpc"`
 
 	TokenManager *TokenManager `yaml:"token_manager"`
 	Reva         *Reva         `yaml:"reva"`
 
-	SkipUserGroupsInToken bool          `yaml:"skip_user_groups_in_token"`
-	AuthProvider          string        `yaml:"auth_provider" env:"AUTH_BEARER_AUTH_PROVIDER" desc:"The auth provider which should be used by the service"`
-	AuthProviders         AuthProviders `yaml:"auth_providers"`
+	SkipUserGroupsInToken bool `yaml:"-"`
```

Comment:
This disables fetching groups. When an admin needs the groups he has to start a custom auth-basic service, right? And there is still a fallback when the groups are needed AFAIR, correct?